topic_admin.go

```diff
@@ -0,0 +1,130 @@
+package kazoo
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+var (
+	ErrTopicExists          = errors.New("Topic already exists")
+	ErrTopicMarkedForDelete = errors.New("Topic is already marked for deletion")
+	ErrDeletionTimedOut     = errors.New("Timed out while waiting for a topic to be deleted")
+)
+
+// CreateTopic creates a new kafka topic with the specified parameters and properties
+func (kz *Kazoo) CreateTopic(name string, partitionCount int, replicationFactor int, topicConfig map[string]string) error {
```

Comment:
Nice job! Do you have plan to implement the feature to modify the partition count and replication factor for existing topics?  Or could you please provide some hint to implement this feature?